.github/workflows/pre-submit.actions.yml

```diff
@@ -182,12 +182,21 @@ jobs:
           path: artifact2
 
       - name: Download artifact
+        id: download-artifact
         uses: ./__BUILDER_CHECKOUT_DIR__/.github/actions/secure-download-artifact
         with:
           name: artifact2
           path: path/to/__BUILDER_CHECKOUT_DIR__/artifact2
           sha256: 5b3513f580c8397212ff2c8f459c199efc0c90e4354a5f3533adf0a3fff3a530
 
+      - name: fail check
```

Comment:
I tested this and it still showed up as failed pre-submit in the GitHub UI. I probably made a mistake during my tests.

Reply:
Yeah, I'm playing with it a bit now.

Reply:
actually it still shows in this PR's pre-submit.

Reply:
I think it should work if I use `continue-on-error` and then check the step `outcome`.

Reply:
cool, do you confirm it works?